<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DevicesDialog</class>
 <widget class="QDialog" name="DevicesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Devices Settings</string>
  </property>
  <layout class="QVBoxLayout" name="DeviceDialogLayout">
   <item>
    <widget class="QGroupBox" name="SettingsGroupBox">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QFormLayout" name="SettingsLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="SnapLabel">
        <property name="text">
         <string>Snap Length</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="SnapLayout">
        <item>
         <widget class="QSpinBox" name="SnapSpinBox">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <number>2147483647</number>
          </property>
          <property name="value">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="SnapUnitLabel">
          <property name="text">
           <string>byte</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="DelayTimeLabel">
        <property name="text">
         <string>Delay Time</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="DelayLayout">
        <item>
         <widget class="QSpinBox" name="DelayTimeSpinBox">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <number>2147483647</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="DelayTimeUnitLabel">
          <property name="text">
           <string>ms</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="OptionLabel">
        <property name="text">
         <string>Option</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="PromiscuousCheckBox">
        <property name="text">
         <string>Promiscuous Mode</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="DeviceLabel">
        <property name="text">
         <string>Device</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QVBoxLayout" name="DeviceLayout">
        <item>
         <layout class="QHBoxLayout" name="DevicesDialogLayout">
          <item>
           <widget class="QLabel" name="CurrentDeviceLabel">
            <property name="text">
             <string>Current Device</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="RefreshButton">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Refresh</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="TableWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="tabKeyNavigation">
           <bool>true</bool>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideMiddle</enum>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Description</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Yes</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>DevicesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>DevicesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
